src/lib/utils.js

```diff
@@ -78,7 +78,14 @@ const getTurkUniqueId = () => {
 const getUserId = (data) => {
   const patientId = JSON.parse(data.responses)['Q0']
   jsPsych.data.addProperties({patient_id: patientId, timestamp: Date.now()})
-  console.log("ID", patientId)
+  console.log("PatientID", patientId)
+}
```

Comment:
are these functions still being used?

Reply:
Yeah those are not really used anymore and I deleted them.